jsdoc/templates/docs/index.html.handlebars

```diff
@@ -122,9 +122,6 @@
         </section>
         {{/each}}
     </main>
-    <script>
-        var DOC_DATA = {{json data}};
-    </script>
```

Comment:
I'm guessing it was a debugging tool accidentally left in, but I don't remember for sure.